src/ape/managers/chain.py

```diff
@@ -271,18 +269,6 @@ def poll_blocks(
         Returns:
             Iterator[:class:`~ape.api.providers.BlockAPI`]
         """
-        network_name = self.provider.network.name
-        block_time = self.provider.network.block_time
-        timeout = (
-            (
-                10.0
-                if network_name == LOCAL_NETWORK_NAME or network_name.endswith("-fork")
-                else 50 * block_time
-            )
-            if new_block_timeout is None
-            else new_block_timeout
-        )
-
         if required_confirmations is None:
             required_confirmations = self.network_confirmations
 
```

Comment:
**todo**: On line 279, `block = None`. You can delete it - it is no longer necessary to declare ahead of time like that.